distribution/docs/src/main/jdocs/content/_plugins/identification-plugin-contents.adoc

```diff
@@ -0,0 +1,17 @@
+:type: plugin
+:status: published
+:title: Identification Plugin
+:link: _identification_plugin
+:plugintypes: preingest,postingest
+:summary: Manages IDs on registry metacards.
+
+The Identification Plugin assigns IDs to registry metacards and adds/updates IDs on create and update.
```

Comment:
Maybe. I guess it depends on whether it's likely to trip up someone using the plugin.